dask_image/ndfilters/_conv.py

```diff
@@ -13,17 +13,17 @@
 
 
 @_utils._update_wrapper(scipy.ndimage.filters.convolve)
-def convolve(image,
-             weights,
-             mode='reflect',
-             cval=0.0,
-             origin=0):
+def convolve(image, weights, mode="reflect", cval=0.0, origin=0):
     check_arraytypes_compatible(image, weights)
 
     origin = _utils._get_origin(weights.shape, origin)
     depth = _utils._get_depth(weights.shape, origin)
     depth, boundary = _utils._get_depth_boundary(image.ndim, depth, "none")
 
+    if mode == "wrap":
+        boundary = "periodic"
+        mode = "constant"
```

Comment:
I don't think it's necessary to include this line? 

When I comment it out (effectively leaving behind `mode="reflect"`) I don't see a difference in the final result. I have only looked at very simple examples though, so if you can explain it more that would be helpful.

Reply:
It is not necessary at all indeed. The reason I did it is because I assume that  it is (barely) faster than doing reflect because padding with constant values is easier than padding reflectively.

Reply:
Ah, I see, you're probably right about that. 
As a general rule, I think that making changes for performance should happen after a benchmarking script shows an improvement.